<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandEditor</class>
 <widget class="QWidget" name="CommandEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>144</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>294</width>
    <height>144</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>314</width>
    <height>239</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Command Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Enter your keystroke including any modifiers:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="shortcut_field">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_clear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QComboBox" name="buttons_specials_field">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>CurrentKey</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Button11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DPIUp</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DPIDown</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DPICycle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ModeSwitch</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DPIShift</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DPIDefault</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_ok">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
